packages/smooth_app/lib/pages/history_page.dart

```diff
@@ -5,6 +5,8 @@ import 'package:smooth_app/database/dao_product_list.dart';
 import 'package:smooth_app/database/local_database.dart';
 import 'package:smooth_app/pages/product/common/product_list_page.dart';
 
+import 'product/common/product_hist_list_page.dart';
```

Comment:
Please use package imports

Reply:
+ please fix the warnings https://github.com/openfoodfacts/smooth-app/runs/6052904115?check_suite_focus=true